x-pack/plugins/infra/server/lib/host_details/get_services.ts

```diff
@@ -79,12 +79,35 @@ export const getServices = async (
         filter: [
           {
             term: {
-              'metricset.name': 'transaction',
+              'processor.event': 'metric',
             },
           },
           {
-            term: {
-              'metricset.interval': '1m', // make this dynamic if we start returning time series data
+            bool: {
```

Comment:
this clause is to also check for "app"  (system metrics). https://www.elastic.co/guide/en/observability/current/data-model-metrics.html .  some services have no transactions.